[2019-05-29 08:37:17] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:108, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2019-05-29 08:37:17] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2019-05-29 08:37:58] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:108, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2019-05-29 08:37:58] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2019-05-29 08:38:29] console.ERROR: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:108, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /Users/Pawel/PHPUnit/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:database:create","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)"} []
[2019-05-29 08:38:29] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2019-05-29 08:40:04] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8002 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Users/Pawel/PHPUnit/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Users/Pawel/PHPUnit/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2019-05-29 08:40:27] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://127.0.0.1:8002/","method":"GET"} []
[2019-05-29 08:40:27] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-05-29 08:40:27] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"e5b668","_route":"_wdt"},"request_uri":"http://127.0.0.1:8002/_wdt/e5b668","method":"GET"} []
